src/session.ts

```diff
@@ -27,10 +28,12 @@ export function useWalletConnect() {
 }
 
 /** Connect user Wallet */
-export async function connectUser() {
+export async function connectUser(provider?: string) {
   const { getDefaultAccount } = createReachAPI();
   try {
     const account: ReachAccount = await getDefaultAccount();
+    if(provider)
+      account.networkAccount.provider = provider;
```

Comment:
😬 oof -- this is a no-no. The `networkAccount` property comes from reach's `stdlib`; we don't want to add any properties to it since that could break the underlying library (for a reachduck user)